### DISCUSSION TITLE
ERROR messages

### DISCUSSION THREAD

**Discussion by BrianPriestBoy** [abraunegg/onedrive#1286] [Q&A] [answered]
1) I'm new to Ubuntu/Linux (7 weeks). I successfully installed and have been using (all the time!) your OneDrive build.

Thank you for making it and maintaining it. I'd love to donate to the cause, if you can tell me how / what could be helpful.

2) I was able to set up and use this without problems. But recently, as I use the "onedrive --synchronize" command I have been receiving the follow two errors:
```
ERROR: Microsoft OneDrive API returned an error with the following message:
    Error Message:    HTTP request returned status code 403 (Forbidden)
    Error Reason:     Access Denied
    Error Timestamp:  2021-02-11T15:15:38
    API Request ID:   96b5c32c-ad4c-4e07-b27d-cc0f0e44559d
    Calling Function: uploadFileDifferences()
```
```
 ERROR: Microsoft OneDrive API returned an error with the following message:
    Error Message:    HTTP request returned status code 403 (Forbidden)
    Error Reason:     Access Denied
    Error Timestamp:  2021-02-11T15:15:42
    API Request ID:   5303d5dc-396f-476f-a7ee-9d1caf09f6ce
    Calling Function: uploadDeleteItem()
```
What approach do you recommend I use to fix these? It does seem that synchronization is occurring, but I'm not sure what to make of these and what to do about them.

Thank you for your help!

fb

**Comment by abraunegg** [comment, marked as answer]
@BrianPriestBoy 
>I'm new to Ubuntu/Linux (7 weeks). I successfully installed and have been using (all the time!) your OneDrive build.
> Thank you for making it and maintaining it. I'd love to donate to the cause, if you can tell me how / what could be helpful.

Thanks for the offer, but not needed. In terms of Ubuntu, as you are new to Linux, try and test out other Linux platforms as well - dont get 'stuck' in the Ubuntu landscape. Also, as you are using Ubuntu, always build the client from source due to Ubuntu packaging issues. If you prefer the way that Ubuntu does things, switch to maybe Debian for a cleaner experience.

> What approach do you recommend I use to fix these? It does seem that synchronization is occurring, but I'm not sure what to make of these and what to do about them.

You need to speak with whoever is managing your OneDrive account online. You do not have the permissions to update the files, nor delete content. This is nothing that the client can solve or fix for you.

**Comment by BrianPriestBoy** [comment]
Accokeek, MD
February 19, 2021

Got it. Thanks!


Yours in Christ,
The Rev. Father Brian Vander Wel
———————————————
“You[, O Lord,] are good and all-powerful,
caring for each one of us as though [we are]
the only one in your care.”
– St. Augustine, Confessions

43rd Rector of Christ Church
600 Farmington Rd.
Accokeek, MD 20607
(301) 292-5633

copyright 2021
